frontend/src/widgets/BarGraph.js

```diff
@@ -52,7 +52,9 @@ function BarGraph({
     counts.push(dataPoint.count);
   });
 
-  const range = [Math.min(...counts), Math.max(...counts)];
+  // We need to buffer the min range/scale otherwise we will cut off the smallest value.
+  const minRange = Math.min(...counts);
+  const range = [minRange === 0 || minRange - 2 <= 0 ? 0 : minRange - 2, Math.max(...counts) + 2];
```

Comment:
is there a case where we wouldn't just want

```js
const range = [0, Math.max(...counts)+2];

```

Reply:
I mean, as the person who wrote this, I can tell you that we did not have that in mind at all